MAUI/Autocomplete/Searching-Filtering.md

```diff
@@ -264,15 +264,15 @@ public class CityViewModel
 
 {% endtabs %}
 
-Now, create a custom filtering class to apply your filter logic to the `Autocomplete` control by following these steps.
+Now, create a custom filtering class to apply your filter logic to the **Autocomplete** control by following these steps.
 
-**Step 1:** Create a class that derives from the `IAutocompleteFilterBehavior` interface.
+**Step 1:** Create a class that derives from the **IAutocompleteFilterBehavior** interface.
 
 {% tabs %}
 {% highlight C# %}
 
 /// <summary>
-/// Represents a custom filtering behavior for the    `Autocomplete` control. 
+/// Represents a custom filtering behavior for the **Autocomplete** control. 
```

Comment:
this is in code section. Revert the change.

Reply:
Changes addressed